patternlab/styleguide/source/_patterns/01-atoms/decorative-link.twig

```diff
@@ -1,6 +1,6 @@
-<span class="ma__decorative-link">
+<span class="ma__decorative-link{% if decorativeLink.button %} button-link{% endif %}">
   <a
     href="{{decorativeLink.href}}"
     class="js-clickable-link"
-    title="{{ decorativeLink.info }}">{{decorativeLink.text}}&nbsp;{{ icon('arrow') }}</a>
+    title="{% if decorativeLink.info %}{{ decorativeLink.info }}{% else %}{{decorativeLink.text}}{% endif %}">{{decorativeLink.text}}&nbsp;{{ icon('arrow') }}</a>

```

Comment:
The readability of this line is not idea, and not to standards.

Since this element is already broken out to multiple lines, I would place the `if`, `else`, & `endif` on separate lines, and not have everything this this super long line.

Additionally, the `{{decorativeLink.text}}&nbsp;{{ icon('arrow') }}` should be on it's own line, and the closing `</a>` also on its own line.